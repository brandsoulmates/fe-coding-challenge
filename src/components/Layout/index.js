import React from "react"

const Layout = ({ children }) => {
  return (
    <>
      <main className="bg-sky-600">
        <div className="mx-auto pt-5 max-w-5xl min-h-screen">
          {children}
        </div>
      </main>
    </>
  )
}

export default Layout
