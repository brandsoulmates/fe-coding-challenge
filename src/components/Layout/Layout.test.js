import React from "react"
import { render } from "@testing-library/react"

import Layout from "."

describe('Layout component', () => {
  it('renders the test correctly', () => {
    const result = render(
      <Layout>
        <div>Layout test</div>
      </Layout>
    )
    expect(result).toMatchSnapshot()
  })
})
