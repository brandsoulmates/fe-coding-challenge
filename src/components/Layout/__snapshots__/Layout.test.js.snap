// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component renders the test correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="bg-sky-600"
      >
        <div
          class="mx-auto pt-5 max-w-5xl min-h-screen"
        >
          <div>
            Layout test
          </div>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="bg-sky-600"
    >
      <div
        class="mx-auto pt-5 max-w-5xl min-h-screen"
      >
        <div>
          Layout test
        </div>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
